src/trio/_core/_run.py

```diff
@@ -2615,18 +2625,17 @@ def unrolled_run(
                         # protocol of unwrapping whatever outcome gets sent in.
                         # Instead, we'll arrange to throw `exc` in directly,
                         # which works for at least asyncio and curio.
-                        # TODO: remove [unused-ignore] when Outcome is typed
-                        runner.reschedule(task, exc)  # type: ignore[arg-type, unused-ignore]
+                        runner.reschedule(task, exc)  # type: ignore[arg-type]
                         task._next_send_fn = task.coro.throw
                     # prevent long-lived reference
-                    # TODO: develop test for this deletion
+                    # There's no regression test checking the necessity of this
```

Comment:
For this and the following, the `TODO`s provide a way to find these via a simple search. Is there a reason not to include that?

Reply:
Ah of course. Not sure why I did this.